scripts/run_ci_tasks.sh

```diff
@@ -74,7 +74,7 @@ if $ANDROID ; then
     if [[ ! -f ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties ]]; then
       cp -np ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties.sample ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties || true
     fi
-
+    
```

Comment:
PRs are generally easier to review if they contain only changes relevant to the PR. When I create a PR, I first take a quick pass through it and eliminate unrelated changes such as this. It takes some effort as more and more editors do automatic re-formatting.